app/services/hyrax/migrator/crosswalk_metadata_service.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal:true
+
+require 'rdf'
+
+module Hyrax
+  module Migrator
+    # Called by the CrosswalkMetadataActor to map OD1 metadata to OD2
+    class CrosswalkMetadataService
+      include RDF
+
+      CONFIG_FILE = 'config/migrator/crosswalk.yml'
+
+      # Given a graph, returns result hash
+      def crosswalk(graph)
+        graph.statements.each do |statement|
+          data = lookup(statement.predicate.to_s)
+          processed_obj = process(data, statement.object)
+          assemble_hash(data, processed_obj)
+        end
+        @result
+      end
+
```

Comment:
Maybe set all other methods below to `private` with the idea that `#crosswalk` is the only method intended to be used by the caller.